CONTRIBUTING.md

```diff
@@ -288,26 +288,30 @@ node seed
 npm run develop
 ```
 
-### Setup freeCodeCamp via Docker
+### Setup freeCodeCamp using Docker
+#### Isolated
+Use this if you just want to work on freeCodeCamp.
 
-You need to have [docker](https://docs.docker.com/install/) and [docker-compose](https://docs.docker.com/compose/install/) installed before executing commands below.
+You'll need to have [docker](https://docs.docker.com/install/) and [docker-compose](https://docs.docker.com/compose/install/) installed before executing commands below.
 
-Create docker-compose.yml from sample file
+Setup:
 ```bash
-cp sample.docker-compose.yml docker-compose.yml
-```
-
-Setup and run project:
-```bash
-# Setup project
 docker-compose run --rm node npm install
 docker-compose run --rm node npm run only-once
 ```
+
+Run:
 ```bash
-# Run project
 docker-compose up
 ```
 
+#### Shared
```

Comment:
Hoping this makes sense.

Reply:
So, if I have this straight in my head, this will expose only the mongo connection inside the image, but not the loopback server?

Reply:
Docker containers could already communicate between each other, this just exposes the port to the host running the containers. This allows users to any mongo client to connect to the db from their workstation/laptop.

You are right, it does not expose the loopback server. We could totally expose that too.